ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastSyncDownloader.java

```diff
@@ -38,6 +39,9 @@
 import org.apache.logging.log4j.Logger;
 
 public class FastSyncDownloader<C> {
+
+  private static final long FAST_SYNC_RETRY_DELAY = 5;
```

Comment:
(nit / optional) - I'd probably keep this constant as a `Duration` for readability (so its easy to see the time unit).